invenio_checks/services/services.py

```diff
@@ -23,25 +33,94 @@ def rebuild_index(self, identity, uow=None):
         raise NotImplementedError()
 
 
+def get_check_config(id_):
+    """Get a check config by id."""
+    check_config = CheckConfig.query.get(id_)
+    if check_config == None:
+        raise ValueError(f"Check configuration with id '{id_}' not found")
+    return check_config
+
+
 class CheckConfigService(RecordService):
-    """Service for managing and check configurations."""
+    """Service for managing check configurations."""
+
+    @unit_of_work()
+    def create(self, identity, data, uow=None, **kwargs):
+        """Create a check configuration."""
+        self.require_permission(identity, "create")
+
+        try:
+            check_config = CheckConfig(
+                community_id=UUID(data["community_id"]),
+                check_id=data["check_id"],
+                params=data.get("params", {}),
+                severity=data.get("severity", "I"),
+                enabled=data.get("enabled", True),
+            )
```

Comment:
minor: we should use a marshmallow schema for validation. For the `community_id` we should though validate here (since it requires resolving the ID and checking if the community exists, etc.).